pkg/pillar/cmd/zedmanager/memorysizemgmt.go

```diff
@@ -5,8 +5,10 @@ package zedmanager
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/lf-edge/eve/pkg/pillar/types"
+	"github.com/sirupsen/logrus"
```

Comment:
In zedmanager you don't need logrus; you can use log.XXX since log is set up before Run is called.

Reply:
Good point, thank you, fixed